docker/Dockerfile.aws

```
@@ -245,7 +245,7 @@ WORKDIR /workspace
 
 # Define commit SHAs as build args to avoid layer invalidation
 ARG LMCACHE_COMMIT_SHA="0db8ae4746a207a72483d716b5f82545a2ead24b"
-ARG VLLM_COMMIT_SHA="b8b302cde434df8c9289a2b465406b47ebab1c2d"
+ARG VLLM_COMMIT_SHA="f71952c1c49fb86686b0b300b727b26282362bf4"
```

Revert this, why are we going back to the old v0.11.0 tag commit BEFORE the pipeline fix for Blackwell? `b8b302cde434df8c9289a2b465406b47ebab1c2d` is the current v0.11.0 tag commit
```suggestion
ARG VLLM_COMMIT_SHA="b8b302cde434df8c9289a2b465406b47ebab1c2d"
```

---

We shouldn't be, bad rebase.